app/src/main/res/values-hu/strings.xml

```diff
@@ -0,0 +1,500 @@
+    <string name="text_broadcasted">Szöveg közvetítve</string>
+    <string name="cmd_theme_title"><![CDATA[theme <név>]]></string>
+    <string name="cmd_theme_help">A megadott témát alkalmazza a Yantra Launcherre. Példa: \'téma Tokyonight\'</string>
+    <string name="specify_theme">Kérlek, adj meg egy téma nevet</string>
+    <string name="only_theme_name_reqd">Érvénytelen parancs használat. A \'theme\' parancs egy paramétert vesz fel: a téma nevét!</string>
+    <string name="launching_custom_theme_designer">[+] Egyedi téma tervező indítása</string>
+    <string name="setting_theme_to">%1$s téma beállítása</string>
+    <string name="theme_not_found">%1$s nem egy ismert téma! Kérlek, használd a \'theme list\' parancsot a témák listájának megtekintéséhez.</string>
+    <string name="customize_your_theme">Téma testreszabása</string>
+    <string name="color_picker">Színválasztó</string>
+    <string name="hex_code">Hexadecimális kód</string>
+    <string name="select_color">Válassz színt</string>
+    <string name="set">Beállítás</string>
+    <string name="enter_6_digit_hex_code_without_hash">Írd be a 6 jegyű hexadecimális kódot (a # jellel)</string>
+    <string name="invalid_hex_code">Érvénytelen hexadecimális kód</string>
+    <string name="apply">Alkalmaz</string>
+    <string name="cmd_time_title">time [utc] [GMT]</string>
+    <string name="cmd_time_help">Jelenlegi helyi dátum és idő megjelenítése. Az utc arg használata az UTC idő megjelenítéséhez. Egy opcionális időeltérési paramétert is megadhat, amelyet hozzáadhat vagy kivonhat az UTC időből. Példák:\nidő\nidő utc\nidő utc +5:30</string>
+    <string name="time_invalid_param">Érvénytelen paraméter megadva. További információért lásd a \'help time\' parancsot.</string>
+    <string name="time_invalid_diff">Érvénytelen időeltérés megadva. Kérlek, adj meg egy megfelelő paramétert. További információért lásd a \'help time\' parancsot.</string>
+    <string name="cmd_timer_title">időzítő [hossz] [üzenet]</string>
+    <string name="cmd_timer_help">Egy visszaszámláló időzítő létrehozására használható az eszközön lévő alapértelmezett alkalmazás használatával. Arg nélküli használata az időzítő alkalmazás indításához szükséges. A hossznak másodpercben kell lennie, és 1-től 86400-ig (24 óra) terjedhet. Opcionálisan követheti egy üzenet sztring a visszaszámlálóban való megjelenítéshez.\nPéldák:\n\'időzítő 60\' beállítja az időzítőt 1 percre\n\'időzítő 3600 Vigyél ki a szemetet!\' beállítja az időzítőt 1 órára, egy megjelenítendő üzenettel.</string>
+    <string name="no_length_provided_launching_timer_app">Nincs megadva hossz. Az időzítő alkalmazás indítása.</string>
+    <string name="parsing_length_string">:: Hossz sztring értelmezése...</string>
+    <string name="invalid_length_string_provided">Érvénytelen hossz sztring megadva. Egy számot kell megadnia 1 és 86400 között.</string>
+    <string name="setting_timer">
+    <![CDATA[=> Időzítő beállítása...]]></string>
```

Comment:
Please edit this line to keep it in a single line

Reply:
Edited :)